src/main/scala/io/iohk/ethereum/jsonrpc/EthService.scala

```diff
@@ -194,43 +201,42 @@ class EthService(
     import io.iohk.ethereum.mining.pow.PowCache._
 
     val blockNumber = appStateStorage.getBestBlockNumber() + 1
-    //todo delete stub, this transaction is only for demo this code will be deleted in next iteration
-    val fakeAddress = 42
-    val privateKey = BigInt(1, Hex.decode("f3202185c84325302d43887e90a2e23e7bc058d0450bb58ef2f7585765d7d48b"))
-    val keyPair = keyPairFromPrvKey(privateKey)
-    val txGasLimit = 21000
-    val txTransfer = 9000
-    val transaction = Transaction(
-      nonce = blockNumber - 1,
-      gasPrice = 1,
-      gasLimit = txGasLimit,
-      receivingAddress = Address(fakeAddress),
-      value = txTransfer,
-      payload = ByteString.empty)
-    val signedTransaction: SignedTransaction = SignedTransaction.sign(transaction, keyPair, None)
-
-    val txList = Seq(signedTransaction)
-    val ommersList = Nil
-    //todo --------------
-    val block = blockGenerator.generateBlockForMining(blockNumber, txList, ommersList, Address(fakeAddress))
-
-    block match {
-      case Right(b) =>
-        Future.successful(Right(GetWorkResponse(
-          powHeaderHash = ByteString(kec256(BlockHeader.getEncodedWithoutNonce(b.header))),
-          dagSeed = seedForBlock(b.header.number),
-          target = ByteString((BigInt(2).pow(256) / b.header.difficulty).toByteArray)
-        )))
-      case Left(err) =>
-        log.error(s"unable to prepare block because of $err")
-        Future.successful(Left(JsonRpcErrors.InternalError))
-    }
+
+    import scala.concurrent.ExecutionContext.Implicits.global
+    implicit val timeout = Timeout(miningConfig.poolingServicesTimeout)
```

Comment:
minor detail: do we need that as a config param? sounds more like it should be a constant in code